fdbserver/RestoreLoader.actor.cpp

```diff
@@ -109,31 +110,31 @@ ACTOR Future<Void> restoreLoaderCore(RestoreLoaderInterface loaderInterf, int no
 }
 
 // Assume: Only update the local data if it (applierInterf) has not been set
-ACTOR Future<Void> handleRestoreSysInfoRequest(RestoreSysInfoRequest req, Reference<RestoreLoaderData> self) {
+void handleRestoreSysInfoRequest(RestoreSysInfoRequest req, Reference<RestoreLoaderData> self) {
 	TraceEvent("FastRestore").detail("HandleRestoreSysInfoRequest", self->id());
 	ASSERT(self.isValid());
 
 	// The loader has received the appliers interfaces
 	if (!self->appliersInterf.empty()) {
 		req.reply.send(RestoreCommonReply(self->id()));
-		return Void();
+		return;
 	}
 
 	self->appliersInterf = req.sysInfo.appliers;
 
 	req.reply.send(RestoreCommonReply(self->id()));
-	return Void();
+	return;
```

Comment:
```suggestion

```

Reply:
Somehow, my local change was not pushed. The new change has changed this.

Reply:
Somehow, this line still shows up.